integration/integration_test.go

```diff
@@ -71,25 +71,18 @@ func TestIntegration(t *testing.T) {
 			name:       "test-data-source",
 			minVersion: "v0.0.0",
 			expectedOutput: map[string]string{
-				"provisioner.arch":                  runtime.GOARCH,
-				"provisioner.id":                    `[a-zA-Z0-9-]+`,
-				"provisioner.os":                    runtime.GOOS,
-				"workspace.access_port":             `\d+`,
-				"workspace.access_url":              `https?://\D+:\d+`,
-				"workspace.id":                      `[a-zA-z0-9-]+`,
-				"workspace.name":                    `test-data-source`,
-				"workspace.owner":                   `testing`,
-				"workspace.owner_email":             `testing@coder\.com`,
-				"workspace.owner_groups":            `\[\]`,
-				"workspace.owner_id":                `[a-zA-Z0-9]+`,
-				"workspace.owner_name":              `default`,
-				"workspace.owner_oidc_access_token": `^$`, // TODO: need a test OIDC integration
-				"workspace.owner_session_token":     `[a-zA-Z0-9-]+`,
```

Comment:
removed all owner related items from the integration test. We already have a separate integration test for `coder_workspace_owner`.